includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -830,6 +839,47 @@ function ( $method ) use ( $non_excludable_methods ) {
+
+	/**
+	 * Returns the shopper's billing country for checkout-time method gating.
+	 * Protected so tests can supply a deterministic country.
+	 *
+	 * @return string Two-letter ISO billing country, or empty when unknown.
+	 */
+	protected function get_billing_country_for_checkout(): string {
+		$customer = WC()->customer;
+
+		return $customer instanceof WC_Customer ? (string) $customer->get_billing_country() : '';
+	}
```

PHPMD (2.15.0)

[error] The method get_billing_country_for_checkout is not named in camelCase. (undefined)

(CamelCaseMethodName)